src/material-experimental/mdc-tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -81,8 +86,13 @@ export class MatTabNav extends _MatTabNavBase implements AfterContentInit {
                * @deprecated @breaking-change 9.0.0 `platform` parameter to become required.
                */
               @Optional() platform?: Platform,
-              @Optional() @Inject(ANIMATION_MODULE_TYPE) animationMode?: string) {
+              @Optional() @Inject(ANIMATION_MODULE_TYPE) animationMode?: string,
+              @Optional() @Inject(MAT_TABS_CONFIG) defaultConfig?: MatTabsConfig) {
     super(elementRef, dir, ngZone, changeDetectorRef, viewportRuler, platform, animationMode);
+    this.disablePagination = defaultConfig && defaultConfig.disablePagination != null ?
```

Comment:
Isn't this one handled by the base class already?

Reply:
Unfortunately not, it extends `MatPaginatedTabHeader` which doesn't handle the config. You may be thinking of the `_MatTabGroupBase` which does handle this